pkg/machinery/kernel/kernel.go

```diff
@@ -34,5 +41,12 @@ type Param struct {
 
 // Path returns the path to the systctl file under /proc/sys.
 func (prop *Param) Path() string {
-	return path.Join("/proc/sys", strings.ReplaceAll(prop.Key, ".", "/"))
+	res := strings.ReplaceAll(prop.Key, ".", "/")
+
+	// fallback to the old behavior if the key path is not absolute
+	if !strings.HasPrefix(res, "/") {
```

Comment:
why do we need this still? to keep backwards compatibility with other users of machinery? or do we have somewhere old way?

Reply:
bumping this question, as I think the interface should be straightforward. we can keep old methods if we want to, but I would rather explicitly break backwards compatibility by e.g. renaming the fields in the `Param` structure

Reply:
I wonder if we actually have any other users of `machinery`. It looks like only `kernel.Write/Read/DeleteParam` is using that code. Only if that's some third party projects, but do you think it can be the case?